src/import-csv.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import _ from 'lodash';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...args) {
+  if (!file) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-csv [file] [couchdb-url] [...<csv options> <pairs>]',
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
```

Comment:
This part is copied from the config cli which also throws. Which would you prefer? I quite like throwing since the promise should still do a catch.